charts/opensearch/README.md

```diff
@@ -9,19 +9,53 @@ This chart adds some slight modifications so that we can define security config
 
 We also setup IRIS-IAM authentication for opensearch dashboards.
 
-## Prerequisites
+# Prerequisites
 
-### Storage
+## Storage
 We've tested OpenSearch using `longhorn` as default storage - for quick install, ensure longhorn is deployed and available to use. Other storage classes are available and should work - but these haven't been tested
 
-### Ingress
+## Ingress
 For Opensearch and Opensearch Dashboards to be accessible outside the cluster - we recommend using nginx ingress. Make sure its enabled on your cluster
 
+
+# Installation
+
+
+## Setup Secrets
+
+To use this chart, you need to provide secret values. Follow these steps:
+
+1. Copy the template file:
+   ```bash
+   cp secret-values.yaml.template /tmp/secret-values.yaml
+  chmod 600 /tmp/secret-values.yaml
```

Comment:
This should use SOPS encrypted files now, instead of us manually bringing the secrets in using kubectl apply
Longer term it will allow us to manage the secret centrally too

Reply:
yes of course - but that is done in cloud-deployed-apps - for users outside of cloud who might want this chart - this is the way they would add secrets